server/src/routes/launches/launches.controller.js

```
@@ -0,0 +1,65 @@
+async function httpAddNewLaunch(req, res) {
+  const launch = req.body;
+
+  if (!launch.mission || !launch.rocket || !launch.launchDate
+    || !launch.target) {
+    return res.status(400).json({
+      error: 'Missing required launch property',
+    });
+  }
+
+  launch.launchDate = new Date(launch.launchDate);
+  if (isNaN(launch.launchDate)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)